static/success.html

```diff
@@ -0,0 +1,9 @@
+<div class="container">
+    <div class="checkmark-holder">
+      <div class="checkmark-circle"></div>
+      <div class="checkmark-stem"></div>
+      <div class="checkmark-kick"></div>
+    </div>
+    <div class="message-holder"><span class="message">Congratulations, All your tests passed...!!!</span></div>
```

Comment:
lol the ellipsis makes it sound ominous, can you change it to just something like `All tests passed!`

Reply:
😄 ... cool. Let me do that.